src/screens/RatingsModal/Enjoy.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useCallback } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Trans } from "react-i18next";
+import { Linking, Platform, TouchableOpacity } from "react-native";
+import { add } from "date-fns";
+import useFeature from "@ledgerhq/live-common/lib/featureFlags/useFeature";
+import { Flex, Text, Button } from "@ledgerhq/native-ui";
+import styled from "styled-components/native";
+import { urls } from "../../config/urls";
+import { setRatingsDataOfUserInStorage } from "../../logic/ratings";
+import {
+  ratingsDataOfUserSelector,
+  ratingsHappyMomentSelector,
+} from "../../reducers/ratings";
+import { setRatingsDataOfUser } from "../../actions/ratings";
+import { track } from "../../analytics";
+
+const NotNowButton = styled(TouchableOpacity)`
+  align-items: center;
+  justify-content: center;
+  margin-top: 24px;
+`;
+
+type Props = {
+  closeModal: any;
+};
+
+const Enjoy = ({ closeModal }: Props) => {
+  const ratings = useFeature("ratings");
+
+  const dispatch = useDispatch();
+  const ratingsDataOfUser = useSelector(ratingsDataOfUserSelector);
+  const ratingsHappyMoment = useSelector(ratingsHappyMomentSelector);
+  const goToStore = useCallback(() => {
+    track("RedirectedToStore", { source: ratingsHappyMoment.route_name });
+    Linking.openURL(
+      Platform.OS === "ios" ? urls.applestoreRate : urls.playstore,
+    );
```

Comment:
It would be cool to directly leverage the platform's native APIs to do that in-app (not critical, maybe we can improve on this later)
https://github.com/oblador/react-native-store-review

Reply:
We thought about that and did some discovery, this would bring some issues and the native flow didn't met our expectations, we'll be using it later when the overall quality of the app is better